amber/run_tests.py

```diff
@@ -32,7 +32,9 @@
     'images/write_image2d_r32i.amber',
     'images/write_image2d_rg32i.amber',
     'images/write_image2d_rgba32i.amber',
-    'integer/add_sat_short.amber']
+    'integer/add_sat_short.amber',
+    'integer/clz_long.amber',
+    'integer/clz_short.amber']
```

Comment:
Missing the `clz_int.amber` script that you added?

Reply:
No, those are swiftshader suppressions. Right now swiftshader doesn't support non-32-bit types so only those variants are filtered out.

Reply:
Ah sorry, I should've expanded the diff to get that context.